lib/ASTGen/Sources/ASTGen/DeclAttrs.swift

```diff
@@ -107,8 +107,13 @@ extension ASTGenVisitor {
       switch attrKind {
       case .alignment:
         return self.generateAlignmentAttr(attribute: node)?.asDeclAttribute
+
       case .allowFeatureSuppression:
         return self.generateAllowFeatureSuppressionAttr(attribute: node, attrName: attrName)?.asDeclAttribute
+
+      case .addressableSelf:
+        fatalError("unimplemented")
```

Comment:
Please add `.addressableSelf` to simple attribute `case` 🙏 :
https://github.com/swiftlang/swift/blob/64fb19b7908b2f59882e167d13b88f4bdc2d10bb/lib/ASTGen/Sources/ASTGen/DeclAttrs.swift#L185-L186